Initial addition of files

MikeCamel opened the pull request:
Completed initial set of file additions.

MikeCamel committed:
Minor tweaks

Signed-off-by: MikeCamel <mike@p2ptrust.org>

MikeCamel committed:
Sorted calling new enarx-keepldr via systemctl

Signed-off-by: MikeCamel <mike@p2ptrust.org>

MikeCamel committed:
Clippy changes

Signed-off-by: MikeCamel <mike@p2ptrust.org>

lkatalin commented on src/main.rs:
```diff
@@ -82,31 +82,32 @@ mod models {
+                            let mut iterator = deserializer.into_iter::<serde_json::Value>();
+                            //we're only expecting one command
+                            let json_pair = iterator.next();
+                            //for json_pair in iterator {
```
Probably want to get rid of this comment.

MikeCamel replied:
this code is gone.

lkatalin commented on src/main.rs:
```diff
@@ -82,31 +82,32 @@ mod models {
+                                }
+                                Err(e) => println!("not a useful reply {}", e),
                             }
+                            //  break;
```
And this comment.

MikeCamel replied:
this code is gone.

lkatalin commented on src/keep-manager-tester.rs:
```diff
@@ -16,7 +16,7 @@ fn main() {
     let mut command_new_keep: HashMap<String, String> = HashMap::new();
     command_new_keep.insert("command".to_string(), "new-keep".to_string());
-    command_new_keep.insert("keep-arch".to_string(), "wasi".to_string());
+    command_new_keep.insert("keep-arch".to_string(), KEEP_ARCH_WASI.to_string());
```
Where is `KEEP_ARCH_WASI` defined?

MikeCamel replied:
This code has gone (though the answer is "lib.rs"

lkatalin commented on src/main.rs:
```diff
@@ -147,10 +147,9 @@ mod filters {
-        );
+        let operation_type = "daemon";
+        let service_cmd = format!("enarx-keep@{}\\x20{}.service", operation_type, new_kuuid);
+        //let service_cmd = format!("enarx-keep@\"{}\\x20{}\".service", operation_type, new_kuuid);
```
Also an extra comment here.

MikeCamel replied:
This code has gone.

lkatalin reviewed (commented):
The small changes look reasonable compared to what was there before. However, there were some extra comments left in the code that seem like they should be removed.

Also, what are the "file additions" referred to in the PR description?

MikeCamel committed:
Changes to reflect new data model.

Signed-off-by: MikeCamel <mike@p2ptrust.org>

MikeCamel committed:
Making new keep creation/start one phase.

Signed-off-by: MikeCamel <mike@p2ptrust.org>

MikeCamel committed:
Initial keep-manager implementation.

Creates new keeps on requests from keep-manager-tester.
Doesn't list available backends yet.

Signed-off-by: MikeCamel <mike@p2ptrust.org>

MikeCamel committed:
Added some docs and clippy requirement

Signed-off-by: MikeCamel <mike@p2ptrust.org>

Removed static README in favour of dynamically created one

Signed-off-by: MikeCamel <mike@p2ptrust.org>

Added cargo-generated README.md

Signed-off-by: MikeCamel <mike@p2ptrust.org>

MikeCamel commented:
@npmccallum - I'd appreciate your input on the clippy fail, please.  Not sure if the code needs fixing (not convinced), or if we need to change the clippy settings.

MikeCamel committed:
Resolved async issue causing clippy errors

Signed-off-by: MikeCamel <mike@p2ptrust.org>

MikeCamel committed:
Reverting to tokio 0.2

tokio 0.3 not yet supported by warp

Signed-off-by: MikeCamel <mike@p2ptrust.org>

MikeCamel committed:
Removed outdated testing framework

Signed-off-by: MikeCamel <mike@p2ptrust.org>
